new-log-viewer/src/services/LogFileManager.ts

```diff
@@ -152,6 +153,36 @@ class LogFileManager {
         this.#decoder.setDecoderOptions(options);
     }
 
+    /**
+     * Loads log events in the range [`logEventIdx`, `logEventIdx + EXPORT_LOGS_CHUNK_SIZE`) or all
+     * remaining log events if `EXPORT_LOGS_CHUNK_SIZE` log events aren't available.
+     *
+     * @param logEventIdx
+     * @return An object containing the log events as a string.
+     * @throws {Error} if any error occurs when decoding the log events.
+     */
+    loadChunk (logEventIdx: number): {
+        logs: string,
+    } {
+        const logEventEndIdx = Math.min(logEventIdx + EXPORT_LOGS_CHUNK_SIZE, this.#numEvents);
```

Comment:
Let's rename this as `logEventEndNum`.

Reply:
It would be a bit weird to have beginLogEventEndIdx and endLogEventNum, right?

Reply:
Remind me, are the end nums inclusive? endIdx is supposed to be exclusive iirc.

Reply:
> Remind me, are the end nums inclusive?

Right

> endIdx is supposed to be exclusive

Right. Sorry for missing this. 

I think it's fine to use the name `endLogEventIdx` then if we're going to rename the input parameter.

Reply:
Plus, according to our naming guide, Idx is for 0-based indexing variable, which is in this case.

I'll rename it to `endLogEventIdx`, corresponding to `beginLogEventIdx`.

Reply:
Ah, sorry, I already made the change.